shift-shift.go

```diff
@@ -33,13 +33,15 @@ type Message struct {
 
 func main() {
 	var (
-		listDevices  = flag.Bool("list", false, "list all devices listened by evdev")
-		printMode    = flag.Bool("print", false, "print pressed keys")
-		quietMode    = flag.Bool("quiet", false, "be silent")
-		deviceMatch  = flag.String("match", "keyboard", "regexp used to match keyboard device")
-		keysymFirst  = flag.String("first", "LEFTSHIFT", "key used for switcing on first xkb group")
-		keysymSecond = flag.String("second", "RIGHTSHIFT", "key used for switcing on second xkb group")
-		scanOnce     = flag.Bool("scan-once", false, "scan for devices only at startup (less power consumption)")
+		listDevices   = flag.Bool("list", false, "list all devices listened by evdev")
+		printMode     = flag.Bool("print", false, "print pressed keys")
+		quietMode     = flag.Bool("quiet", false, "be silent")
+		deviceMatch   = flag.String("match", "keyboard", "regexp used to match keyboard device")
+		keysymFirst   = flag.String("first", "LEFTSHIFT", "key used for switching on first xkb group")
+		keysymSecond  = flag.String("second", "RIGHTSHIFT", "key used for switching on second xkb group")
+		scanOnce      = flag.Bool("scan-once", false, "scan for devices only at startup (less power consumption)")
+		dblKeystroke  = flag.Bool("double-keystroke", false, "require pressing the same key twice to switch the layout")
+		dblKeyTimeout = flag.Int("second-keystroke-timeout", 500, "second keystroke timeout in milliseconds")
```

Comment:
It would be better to stick to flag prefix: `double` or `second`

Reply:
Agreed. Previously it has same prefixes, I'm really uncertain when it comes to naming :\